tests/utils.go

```diff
@@ -2675,3 +2675,61 @@ func dvSizeBySourceURL(url string) string {
 
 	return cd.CirrosVolumeSize
 }
+
+func GetSnapshotStorageClass(client kubecli.KubevirtClient) (string, error) {
```

Comment:
I think we dont want to add more logic to tests/utils. Maybe:
tests/libstorage/storageclass.go
Is a better place

Reply:
+1 For not adding more stuff to `utils.go`

Reply:
done